### PR TITLE
Change lmp to weeks along in patient edit view

### PR DESCRIPTION
**Pull request by colinmcglynn**
I rule and have completed some work on Case Manager that's ready for review!

This issue changes the views were necessary to show 'Weeks along at intake' instead of 'LMP at intake'.

Equally important are where 'lmp' was not changed
* variable and function names
* test descriptions
* the description in DATA_DICTIONARY
* anything in the jekyll_artifacts directory (is this defunct?)

That last one probably should change, but I don't want to get the verbiage wrong. @lwaldsc could you help me out with that?

It relates to the following issue #s: 
* Fixes https://github.com/DCAFEngineering/dcaf_case_management/issues/453

screen shot 2017-09-09 at 2 25 44 pm

screen shot 2017-09-09 at 2 26 59 pm

### CODE REVIEW

File: docs/DATA_DICTIONARY.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Field | Description | Exportable? | Reported by | Input Type
...
-**LMP at intake** | How many weeks from a patient's Last Missed Period as reported during our first interaction. This is often a guess from the patient, but may also be an exact calculation from an ultrasound. We use LMP interchangably with 'weeks pregnant' in our funding contexts, even though this may be a different number in actuality. While DARIA automatically calculates LMP after this first interaction, LMP is editable if a patient receives a more accurate calculation later in the case management process. | Exportable | Patient Reported | Text field
+**Weeks along at intake** | How many weeks from a patient's Last Missed Period as reported during our first interaction. This is often a guess from the patient, but may also be an exact calculation from an ultrasound. We use LMP interchangably with 'weeks pregnant' in our funding contexts, even though this may be a different number in actuality. While DARIA automatically calculates LMP after this first interaction, LMP is editable if a patient receives a more accurate calculation later in the case management process. | Exportable | Patient Reported | Text field
```
Comment (lomky) [inline docs/DATA_DICTIONARY.md]: I suggest introducing the LMP acronym in parenthesis after the first time we write Last Missed Period. Like: "...Last Missed Period (LMP)".

For the rest of the wording, I second asking @lwaldsc for help!

**Comment by colinxfleming** [comment]
Jekyll artifacts is in fact EXTREMELY defunct! We should nuke that entire dir, as we are an app now instead of a series of mockups.

LGTM from a code perspective, let's merge after we get an all clear on the language from @lwaldsc .

**Comment by lwaldsc** [comment]
Just wondering what's the reasoning behind changing it to "Weeks along"
rather than LMP. I'm sorry if this is obvious, but just want to make sure
I'm on the same page.

**Comment by colinmcglynn** [comment]
@lwaldsc I believe it came out of a conversation you had with @NerdyGirl537 and @eheintzelman? https://github.com/DCAFEngineering/dcaf_case_management/issues/453#issuecomment-230945073

**Pull request merged**